docs/01_setup.md

```diff
@@ -83,31 +90,35 @@ Copy the config example and update the fields as necessary:
 cp deploy-config-example.json deploy-config.json
 ```
 
-There are already some deployed contracts that could be reused, copy the `default-deployed-contracts.json` file if you need them to avoid deploying redundant contracts and save your gas fee.
+> [!IMPORTANT]
+> Make sure that you use the production zkeys (copy the correct generated coordinatorPubkey), set the `pollDuration` with the correct time in **seconds** and set `useQuadraticVoting` to false.
```

Comment:
You refer to the coordinator maci key instead of the production keys?
MACI Platform now supports quadratic voting, so that depends on the round

Reply:
I think we should clarify the zkeys vs coordinatorPubKey?
the production zkeys refer to the zkeys for the VkRegistry, meanwhile the coordinatorPubKey is generated through the maci-cli, which is used for deploying the Poll contract, they are different things.

Reply:
Should I leave then:
> [!IMPORTANT]
> Make sure that you replace the coordinatorPubkey generated before in a [previous step](#Generate-MACI-Keys), and set the `pollDuration` with the correct time in **seconds**